features/ubuntu_pro.feature

```diff
@@ -185,8 +185,8 @@ Feature: Command behaviour when attached to an UA subscription
             cis           +yes  +<cis-s>  +Center for Internet Security Audit Tools
             esm-apps      +yes +enabled +UA Apps: Extended Security Maintenance \(ESM\)
             esm-infra     +yes +enabled +UA Infra: Extended Security Maintenance \(ESM\)
-            fips          +yes +n/a +NIST-certified FIPS modules
```

Comment:
I think fips status here should be `n/a` since we have a `static_affordance` to block enabling any fips service on GCP.